kuma/payments/jinja2/payments/includes/payments-form.html

```diff
@@ -1,4 +1,9 @@
-<form id="contribute-form" class="contribution-form" method="{% if recurring_payment and not request.user.is_authenticated %}get{% else %}post{% endif %}" action="{% if recurring_payment %}{{ recurring_payment_subscription_url }}{% else %}{{ payments_url }}{% endif %}" novalidate>
+<form id="contribute-form"
+    class="contribution-form"
+    method="{% if recurring_payment and not request.user.is_authenticated %}get{% else %}post{% endif %}"
+    action="{% if recurring_payment %}{{ recurring_payment_subscription_url }}{% else %}{{ payments_url }}{% endif %}"
+    data-payment-type="{% if recurring_payment %}recurring{% else %}one-time{% endif %}"
+    novalidate>
```

Comment:
I find all this logic confusing, would like to refactor this at a later stage. I've created a task in the backlog to look at this.

Reply:
The new layout makes it less confusing than it was before, nice change.